docs/user-docs/latest/content-types-builder/configuring-fields-content-type.md

```diff
@@ -0,0 +1,410 @@
+| Unique field   | Tick to prevent another field to be identical to this one.                  |
+| Maximum length | Tick to define a maximum number of characters allowed.                      |
+| Minimum length | Tick to define a minimum number of characters allowed.                      |
+
+:::
+
+::::
+
+### <img width="28" src="../assets/content-types-builder/field-icon_date.png"> Date
+
+The Date field can display a date (year, month, day), time (hour, minute, second) or datetime picker.
+
+:::: tabs
+
+::: tab Base settings
+
+| Setting name  | Instructions                                                    |
+|---------------|-----------------------------------------------------------------|
+| Name          | Write the name of the Date field.                               |
+| Type          | Choose between *date*, *datetime* and *time*                    |
+
+:::
+
+::: tab Advanced settings
+
+| Setting name   | Instructions                                                                |
+|----------------|-----------------------------------------------------------------------------|
+| Default value  | Write the default value of the Date field.                                  |
+| Private field  | Tick to make the field private and prevent it from being found via the API. |
+| Required field | Tick to prevent creating or saving an entry if the field is not filled in.  |
+| Unique field   | Tick to prevent another field to be identical to this one.                  |
+
+:::
+
+::::
+
+### <img width="28" src="../assets/content-types-builder/field-icon_boolean.png"> Boolean
+
+The Boolean field displays a toggle button to manage boolean values (e.g. Yes or No, 1 or 0, True or False).
+
+:::: tabs
+
+::: tab Base settings
+
+| Setting name  | Instructions                                                    |
+|---------------|-----------------------------------------------------------------|
+| Name          | Write the name of the Boolean field.                            |
+
+:::
+
+::: tab Advanced settings
+
+| Setting name   | Instructions                                                                |
+|----------------|-----------------------------------------------------------------------------|
+| Default value  | Write the default value of the Boolean field.                               |
+| Private field  | Tick to make the field private and prevent it from being found via the API. |
+| Required field | Tick to prevent creating or saving an entry if the field is not filled in.  |
+| Unique field   | Tick to prevent another field to be identical to this one.                  |
+
+:::
+
+::::
+
+### <img width="28" src="../assets/content-types-builder/field-icon_relation.png"> Relation
+
+The Relation field allows to establish a relation with another content-type, that must be a collection type.
+
+There are 6 different types of relations:
+
+- Content-type A *has one* Content-type B
```

Comment:
We should clarify these with their developer names, in order:

- oneWay
- oneToOne
- oneToMany
- manyToOne
- manyToMany
- manyWay

Reply:
(we use these names quite often) Also wondering if an image of the relation matrix would be useful here?

Reply:
Do I have to write the relation types names exactly like that @derrickmehaffy or can I write for example "one way", "one-to-one"? 🤔 
And you're right I'll add the images it could be quite useful!

Reply:
Shouldn't matter as long as the context makes sense, we use that name structure as it's commonplace among programmers and developers.

But one-to-one vs oneToOne shouldn't matter. Whatever looks best to you.